PNChart/PNPieChart.m

```diff
@@ -124,7 +124,8 @@ - (UILabel *)descriptionLabelForItemAtIndex:(NSUInteger)index{
     UILabel *descriptionLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, 100, 60)];
     NSString *titleText = currentDataItem.textDescription;
     
-    NSMutableAttributedString* attributedTitleText = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"%.0f%%\n",currentDataItem.value/ _total * 100] attributes:@{NSForegroundColorAttributeName:currentDataItem.color,NSFontAttributeName:_fontForDetailItemNumber}];
+    int normalizedValue = (_total == 0 ? 0 : currentDataItem.value/ _total) * 100;
```

Comment:
the parenthesis are better to separate the first `=` to the next `==` that could be confusing